package.json

```diff
@@ -33,7 +33,7 @@
   "author": "Didier Baquier",
   "license": "MIT",
   "bugs": {
-     "url": "https://github.com/cordova-sms/cordova-sms-plugin/issues"
+     "url": "https://github.com/cordova-sms/com.cordova.plugins.sms/issues"
   },
-  "homepage": "https://github.com/cordova-sms/cordova-sms-plugin#readme"
+  "homepage": "https://github.com/cordova-sms/com.cordova.plugins.sms#readme"
```

Comment:
@tajinder-logiciel I don't think, changing homepage url is good idea, it will return 404.

Reply:
@MayankLogiciel URL changed, also that previous URL is not our repo's so I did change this to - https://github.com/MayankLogiciel/cordova-plugin-mms-ios/issues